io/include/pcl/io/boost.h

```diff
@@ -83,5 +82,9 @@
 #include <boost/signals2/slot.hpp>
 #endif
 #endif
+#ifndef Q_MOC_RUN
+#include <boost/algorithm/string.hpp>
+#include <boost/interprocess/sync/file_lock.hpp>
```

Comment:
You need to erase the copy of this in line 66.

Reply:
Fixed